lib/test.dart

```diff
@@ -8,6 +8,7 @@ import 'package:path/path.dart' as p;
 
 import 'src/backend/declarer.dart';
 import 'src/backend/invoker.dart';
+import 'src/backend/platform_selector.dart' show PlatformSelector; // for the dartdocs
```

Comment:
Dartdoc these days assumes everything it knows about is in scope for every dartdoc reference. That way we can have documentation at low layers (e.g. in `dart:ui`) that reference higher layers (e.g. `package:flutter`) which it otherwise couldn't import, either for technical reasons (like in the `dart:ui`->`package:flutter` case) or for policy reasons (e.g. we don't want it to be possible for code from the widgets layer to be in scope at the rendering layer, since that would make it too easy to introduce bidirectional dependencies in the implementation).